apps/api/src/server/applications/application/__tests__/get-application-details.test.js

```diff
@@ -6,12 +6,16 @@ import { applicationFactoryForTests } from '../../../utils/application-factory-f
 import { databaseConnector } from '../../../utils/database-connector.js';
 
 const request = supertest(app);
+const clock = new Date().toISOString();
```

Comment:
can we return the dates in a Number format rather than ISO Strings? It is what other APIs do and what the frontend expects.

You can use the key dates as a reference